src/models/GitWalkerFs.js

```diff
@@ -102,11 +98,7 @@ export class GitWalkerFs {
         const config = await GitConfigManager.get({ fs, gitdir })
         const autocrlf = await config.get('core.autocrlf')
         const content = await fs.read(`${dir}/${entry._fullpath}`, { autocrlf })
-        // workaround for a BrowserFS edge case
         entry._actualSize = content.length
-        if (entry._stat && entry._stat.size === -1) {
-          entry._stat.size = entry._actualSize
-        }
```

Comment:
The same please don't remove code for BrowserFS compatibility from the codebase, we will need to release version 2.0 for no reason.

Reply:
Got it.

Reply:
https://github.com/isomorphic-git/isomorphic-git/pull/1996/commits/7ccd61703d9a2a85c04d32f5320752e11fc215e7